tests/unit/phpDocumentor/Descriptor/Tag/LinkDescriptorTest.php

```diff
@@ -17,6 +17,12 @@
 
 /**
  * Tests the functionality for the LinkDescriptor class.
+ *
+ * @uses \phpDocumentor\Descriptor\Collection
+ *
+ * @coversDefaultClass \phpDocumentor\Descriptor\Tag\LinkDescriptor
+ * @covers \phpDocumentor\Descriptor\TagDescriptor::__construct
```

Comment:
You could replace this with `@covers \phpDocumentor\Descriptor\TagDescriptor`; usually I am non-specific when it comes to covering abstract/parent classes as this reduces administration and is recommended by Sebastian Bergmann